@xen-orchestra/web/src/components/site/pools/PoolSidePanel.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+  <VtsLoadingHero v-if="!isPoolReady && !isHostready" type="panel" />
```

Comment:
```suggestion
  <VtsLoadingHero v-if="!isPoolReady" type="panel" />
```

I don't think `isHostReady` is necessary here. If there is no host, this means that we are not displaying the sidepanel.